isaric/parsers/ccp-netherlands.toml

```diff
@@ -310,6 +310,7 @@
       { field = "patient_interventions#Highflow_nasal_canula_oxygen_therapy", values = { 1 = true, 0 = false } },
       { field = "patient_interventions#Extra_corporeal_life_support_ECLS", values = { 1 = true, 0 = false } },
       { field = "Extracorporeal_support", ref = "Y/N" },
+      { field = "fio2_1", if = { fio2_1 = { ">" = 21 } }, apply = { function = "isNotNull" } },
     ]
```

Comment:
the variable signssymp_L_oxygen also gives L oxygen, I think at admission although not super clear! This can also  be mapped to the oxygen flow attribute with description 'amount of oxygen therapy (L)'